docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -25,42 +25,62 @@ This tutorial will show you how to build a mock robot using just your personal l
 - Learn how to [configure](../../../manage/configuration/) robots with Viam.
 - Try using [Viam](/viam/) without any robotic hardware.
 
-Most Viam [components](../../../components/) have a _fake_ model that you can use for testing.
-These fake components interact with Viam like real hardware but do not actually exist.
+Follow this tutorial to set up and control a robot with a `fake` [arm](/components/arm/fake/), [board](/components/board/), and [motor](/components/motor/), and an additional mock {{< glossary_tooltip term_id="part" text="sub-part" >}} with a [motor](/components/motor/).
 
-In this tutorial, you will set up, control, and program a mock robotic arm and a mock sub-part with a motor using fake components.
+### What is a mock sub-part?
```

Comment:
Moved this info up to the top vs in middle of page so users have this context immediately if they want to learn about it